### DISCUSSION TITLE
Contribution heatmap since 1970

### DISCUSSION THREAD

**Discussion by IliaOzhmegov** [community/community#34992] [Repositories] [answered]
Hi folks,

Not sure if it is supposed to submitted here. Still I'll try. I have generated a contribution heatmap at 1970 at private repo just to see if it's possible and it worked! The problem is that when I removed a repository that was responsible for contributions at 1970 the button between current year and 1970 are left. You can see it at my page https://github.com/IliaOzhmegov.

<img width="163" alt="Screenshot 2022-10-03 at 22 32 23" src="https://user-images.githubusercontent.com/43893994/193676797-62e401b0-c8a7-4429-83be-1ea9912bc5fb.png">

Please let me know how can it be fixed or where am I supposed to submit this issue?

Thanks in advance!

**Comment by radar-blip** [comment, marked as answer]
I remember someone posting a similar situation. I believe you will have to contact support to fix it for you.
Here is a link for that,  maybe under  _remove data from a repository_ [Support](https://support.github.com/request)

Or you could try filing a [Bug-Report](https://support.github.com/contact/bug-report)

**Comment by IliaOzhmegov** [comment]
Hi @linuxlawson,

Thank for a reply! I'll try to reach out Support and try to fill in Bug-report. If any those got answered I'll try to keep the discussion updated on that.

**Comment by IliaOzhmegov** [comment]
I did both, and GH automatically merge two tickets into one! Let's see what will happen next :)

**Comment by IliaOzhmegov** [comment]
The support solved it! 
It happened because:
`We automatically keep backup of deleted repositories for 90 days before they are auto-purged, so there is no way for our users to purge them, but they can write in and request it and we can do it.`
Now it's purged and everything is okay.

**Comment by radar-blip** [comment]
Awesome! Glad you were able to get it fixed! 😃
